### ISSUE TITLE
F_theta - F_theta_minus is zero,Please confirm where the bug is?

### ISSUE THREAD

**Issue by wisestruct** [xandergos/sCM-mnist#2]
@ loss = (weight / torch.exp(logvar)) * torch.square(F_theta - F_theta_minus - g) + logvar

**Comment by xandergos** [comment]
This is not a bug. F_theta_minus is just F_theta.detach() so the values are always the same. This is not an MSE loss like it is usually used in ML, instead its just a neat trick to rewrite a different type of loss with MSE. See the paper for details.
